tests/unit/batcomputer_cli/commands/test_identify.py

```diff
@@ -1,10 +1,28 @@
+import string
 import pytest
 
 from ....batcomputer_cli.commands.identify import identify
 from ....batcomputer_cli.database import CHARACTERS
 
 NON_IDENTIFIED_CHARACTERS = ["riddler", "nightwing", "penguin"]
 
+CHARACTER = list(CHARACTERS)[0]
```

Comment:
here instead of creating a new test, what about using the already existing test?

We could do something like
```python
UPPER_CASE_CHARACTERS = [
    character.upper()
    for character in CHARACTERS
]
TITLE_CASE_CHARACTERS = [
    character.title()
    for character in CHARACTERS
] # This does the same as string.capwords, right?


@pytest.mark.parametrize("alias", list(CHARACTERS)+UPPER_CASE_CHARACTERS+TITLE_CASE_CHARACTERS)
def test_identify_with_one_identified_alias(
```

Reply:
Since the body of the tests are the same, I feel it is duplication to have separate tests

Reply:
Let me know if I should combinate the test as I fixed `get_oracle` as well!

Reply:
I think separate how is now is better :smile: 

Fixing CI should be enough to close this PR, nice work and thank you very much!